code/modules/client/preference_setup/loadout/lists/uniforms.dm

```diff
@@ -173,19 +173,9 @@
 	gear_tweaks += new /datum/gear_tweak/path(stylish)
 
 /datum/gear/uniform/dress
-    display_name = "dresses"
-    path = /obj/item/clothing/under/dress
-
-/datum/gear/uniform/dress/New()
-	..()
-	var/dress = list(
-		"Gray Dress"			=	/obj/item/clothing/under/dress,
-		"Blue Dress"			=	/obj/item/clothing/under/dress/blue,
-		"Red Dress"				=	/obj/item/clothing/under/dress/red,
-		"White Dress" 			=	/obj/item/clothing/under/dress/white,
-		"Black Dress" 			=	/obj/item/clothing/under/dress/black
-	)
-	gear_tweaks += new /datum/gear_tweak/path(dress)
+	display_name = "dress selection"
+	path = /obj/item/clothing/under/plaid
```

Comment:
this meant to be plaid?

Reply:
Yes. I don't recall exactly why I originally used that path but that's correct IIRC. You did however remind me, I need to adjust some pathing.

Reply:
Nevermind. You were right, had to do more than just repath a few things- whoever did this is not going to heaven.